spaceapi_server/Cargo.toml

```diff
@@ -10,6 +10,7 @@ authors = [
 redis = "0.3"
 rustc-serialize = "0.3"
 iron = "0.1"
+urlencoded = "*"
```

Comment:
I generally wouldn't pin anything to minor versions since crates should use semantic versioning. Please adjust pin to be `urlencoded = "^0.2"`. I'd also specify the other crates versions with caret.

At Renuo we often even don't pin to major versions (but carefully supervising the *lock*-files). `Cargo.lock` is exactly there for being specific in what you use but being free in what you need. This allows you to keep your app up-to-date without much risk.